docs/release/trg-0/trg-4-02.md

```diff
@@ -0,0 +1,156 @@
+|---------------------------|----------------------------------------------------------------------------|--------------------------------------------------------------------------------------------------------------------------------|
+| Java / Kotlin / JVM based | [Eclipse Temurin](https://hub.docker.com/_/eclipse-temurin)                | prefer JRE over JDK and alpine tags for your JRE version, e.g.:<br/>`21-jre`, `21-jre-alpine`, `20-jre`, `20-jre-alpine`, etc. |
+| JS frontends              | [nginx-unprivileged](https://hub.docker.com/r/nginxinc/nginx-unprivileged) | prefer `1.25`, `1.25-alpine` or `alpine` tag                                                                                   |
+| .NET runtime              | [.NET runtime](https://hub.docker.com/_/microsoft-dotnet-runtime)          | prefer `8.0-alpine` tag                                                                                                        |
+| ASP.NET runtime           | [ASP.NET core runtime](https://hub.docker.com/_/microsoft-dotnet-aspnet)   | prefer `8.0-alpine` tag                                                                                                        |
+| Linux                     | [Alpine Linux](https://hub.docker.com/_/alpine)                            | prefer `3.16`, `3.17`, `3.18` or `3.19` tag                                                                                    |
+
+If the language or runtime environment of your product is not listed above, feel free
+to [start a discussion](https://github.com/eclipse-tractusx/sig-infra/discussions) and propose your preferred container
+images as base image.
+
+:::info
+As stated in the description above, base image usage is particularly aligned for container images, that we distribute by
+publishing them on DockerHub. In case you are using Docker images for build or testing purposes (for example _pandoc_ or
+_cypress_, etc.) and you do not publish the images, you can use other publicly available image, as long as the tools are
+open source license compliant.
+
+For automated TRG checks, you can skip base image checks on Dockerfiles by declaring it in the `.tractusx` metadata
+files. Details can be found in
+the [metadata file documentation](https://github.com/eclipse-tractusx/tractusx-quality-checks/blob/main/docs/metadata_file.md)
+:::
+
+### Use minor or major Image Tag
+
+Do not use an image tag pointing to a patch version of the image. Instead, use an image tag containing either minor or
+major version.
+
+To keep application images compliant, up-to-date and secure it is mandatory to use image tags which allows to rebuild
+the applications container image without touching the `DOCKERFILE` to pull an updated version of the base image (e.g. to
```

Comment:
`Dockerfile` is usually not written in caps.

Reply:
Fixed with latest commit.